src/lib/Components/Bidding/Components/__tests__/MarkdownRenderer-tests.tsx

```diff
@@ -1,59 +1,84 @@
-import { shallow } from "enzyme"
+// import { shallow } from "enzyme"
 import React from "react"
 import "react-native"
 import * as renderer from "react-test-renderer"
+import { BiddingThemeProvider } from "../BiddingThemeProvider"
 import { MarkdownRenderer } from "../MarkdownRenderer"
 
 describe("Markdown parser", () => {
-  it("renders text", () => {
+  it("renders text only", () => {
     const md = "text without link"
-    const markdown = shallow(<MarkdownRenderer md={md} />)
+    const component = (
+      <BiddingThemeProvider>
+        <MarkdownRenderer>{md}</MarkdownRenderer>
+      </BiddingThemeProvider>
+    )
 
-    expect(markdown.find("Text")).toHaveLength(1)
-    expect(markdown.find("Text").prop("children")).toMatch("text without link")
+    // const markdown = shallow(<MarkdownRenderer>{md}</MarkdownRenderer>)
```

Comment:
Ditto to @ashfurrow can we mark these as `xit` if we want to skip them and remove the `console.log`s?